ROADMAP.md

```diff
@@ -0,0 +1,46 @@
+# Declarative Gradle Roadmap
+
+Declarative Gradle is an experimental project under active development.
+Below, you can see the key milestones we anticipate.
+All the dates and the scope are aspirational.
+
+To  discuss the roadmap and the related initiatives, use the
+`#declarative-gradle` channel on the [Gradle Community Slack](https://gradle.org/slack-invite)
+
+## Project Board
+
+We have a [project board](https://github.com/orgs/gradle/projects/96) published on GitHub Project.
+It captures the main initiatives and EPICs in this project.
+
+## Key Milestones
+
+### First Alpha Release
+
+Scope: Declarative Gradle can be evaluated with limited functionality and with support from the team.
+This includes:
+the alpha release of the Declarative Gradle plugins,
+basic integrations, demos, and documentation.
+In this release, our primary focus is Android development,
+but we keep in mind classic Java and Kotlin use cases,
+as well as Kotlin Multiplatform (KMP).
+
+- [Planned Issues (milestone)](https://github.com/gradle/declarative-gradle/milestone/1)
+- [Prototypes](./unified-prototype/README.md)
+
+### Public Early Access Release (EAP)
+
+The first version is ready for broad external evaluation and feedback.
+That includes making the Declarative Gradle plugins functional for core use cases and
```

Comment:
This is unclear to me.  Is this trying to say that saying evaluation includes making the plugins function for core use cases?  Or that the plugins are already functional as of the EAP and are ready for evaluation?  I think the first of these is truer than the second one.

Reply:
```suggestion
That includes making the Declarative Gradle plugins functional for core use cases, while this work is likely to continue after the milestone, and
```

Added clarification